components/script/textinput.rs

```diff
@@ -987,11 +1118,12 @@ impl<T: ClipboardProvider> TextInput<T> {
         self.assert_ok_selection();
     }
 
+    /// Set the edit point index position based off of a given grapheme cluster offset
```

Comment:
This is correct documentation for what this method currently does, but that behavior is buggy. `index` appears to come from here:

https://github.com/servo/servo/blob/36c75c4e5f5a322b5b21137615b9c41e5f8a2427/components/gfx/text/text_run.rs#L369-L385

… which is counting glyphs. Grapheme count only sometimes matches glyph count.

Fixing this is probably a larger change than should go into this already non-trivial PR.

Reply:
oh huh. Should I add a note to that effect? Can also open an issue.

Reply:
I’ve filed https://github.com/servo/servo/issues/23399.